test/variableTests.js

```diff
@@ -25,7 +26,7 @@ describe('Variable', function() {
     it('should call _parse with the given scssString', function() {
       var parseStub = sinon.stub(Variable.prototype, '_parse');
 
-      new Variable(scssString);
+      new Variable(scssString); // eslint-disable-line no-new
```

Comment:
To pass the linter, I think you can just call `Variable(scssString)`. I believe calling it with new adds the prototype to the object, but since we only use the constructor and never reuse the returned object, we don't need the prototype object.